README.md

```diff
@@ -8,88 +8,42 @@ JavaScript SDK for integrating [Pera Wallet](https://perawallet.app) to web appl
 
 ### Getting Started
 
-[Try it out using CodeSandbox](#example-applications)
-
 [Learn how to integrate with your JavaScript application](#guide)
 
 [Learn how to Sign Transactions](#sign-transaction)
 
+[Try it out using CodeSandbox](#example-applications)
+
 ### Example Applications
 
 [Using React Hooks](https://codesandbox.io/s/perawallet-connect-react-demo-ib9tqt?file=/src/App.js)
 
-[Using Vue3](https://codesandbox.io/s/perawallet-connect-vue-demo-m8q3sl?file=/src/App.vue)
-
-### Guide
+Using React Hooks with React@18 (Will be added soon)
 
-Let's start with installing `@perawallet/connect`
-
-```
-npm install --save @perawallet/connect
-```
+[Using Vue3](https://codesandbox.io/s/perawallet-connect-vue-demo-m8q3sl?file=/src/App.vue) (Will be added soon)
 
-<details>
-  <summary>Using with React 18</summary><br/>
-  
-   When you want to use `@perawallet/connect` library with React 18, you need to make some changes. `react-scripts` stopped polyfilling some of the packages with the `react-scripts@5.x` version. After creating a new app with `npx create-react-app my-app` or in your react application, the following changes should be made.
+[Using Svelte](https://codesandbox.io/s/pera-connect-svelte-demo-zkq6b9)
 
-1. Firstly, install the following packages.
+Using Next.js (Will be added soon)
 
-```sh
-  npm install buffer
-  npm install crypto-browserify
-  npm install process
-  npm install react-app-rewired
-  npm install stream-browserify
-```
+Using Nuxt.js (Will be added soon)
 
-2. After that you need to override some webpack features. Create the following file in the root directory of the project and copy the following code block into it.
+### Quick Start
 
-`config-overrides.js`
+Let's start with installing `@perawallet/connect`
 
-```jsx
-// eslint-disable-next-line @typescript-eslint/no-var-requires
-const webpack = require("webpack");
-
-module.exports = function override(config) {
-  const fallback = config.resolve.fallback || {};
-
-  Object.assign(fallback, {
-    crypto: require.resolve("crypto-browserify"),
-    stream: require.resolve("stream-browserify")
-  });
-  config.resolve.fallback = fallback;
-  // ignore warning about source map of perawallet/connect
-
-  config.ignoreWarnings = [/Failed to parse source map/];
-  config.plugins = (config.plugins || []).concat([
-    new webpack.ProvidePlugin({
-      process: "process/browser",
-      Buffer: ["buffer", "Buffer"]
-    })
-  ]);
-  return config;
-};
+```
+npm install --save @perawallet/connect
 ```
 
-3. Finally, you need to update the npm scripts.
-
-`{ "start": "react-app-rewired start", "build": "react-app-rewired build" }`
-
-After that, you are good to go! 🎊
-
-</details>
-
-#### Using React Hooks
-
-```typescript
+```jsx
```

Comment:
Let's shorten this example a bit, no need for the React details. We already have examples 

```typescript
// Connect handler
peraWallet
  .connect()
  .then((newAccounts) => {
    // Setup the disconnect event listener
    peraWallet.connector?.on("disconnect", handleDisconnectWalletClick);

    setAccountAddress(newAccounts[0]);
  })
  .reject((error) => {
    // You MUST handle the reject because once the user closes the modal, peraWallet.connect() promise will be rejected.
    // For the async/await syntax you MUST use try/catch
    if (error?.data?.type !== "CONNECT_MODAL_CLOSED") {
      // log the necessary errors
    }
  });
```

```typescript
// On the every page refresh 
peraWallet.reconnectSession().then((accounts) => {
  // Setup the disconnect event listener
  peraWallet.connector?.on("disconnect", handleDisconnectWalletClick);

  if (accounts.length) {
    setAccountAddress(accounts[0]);
  }
});
```